### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_Keyboard: Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/AzureStorageExplorer#7623]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1 (96)

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

Windows

### OS Version

Windows 11 Enterprise Insider Preview (23H2)

### GitHub Tags:
#A11yMAS;#A11yTCS;#AILimited;#Win11;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Win32_Dec2023;#Azure Tools Storage Explorer;#FTP;#WCAG2.4.7;#FocusVisible;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab Navigate to "Open Connect dialog" pane and Sign into Azure.
3. Tab Navigate to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Select "Blob Containers" of the selected subscription.
5. Tab Navigate to "Properties" Tab present on left bottom of the screen and hit enter.
6. Observe that Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.

### Actual Experience

Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.

### Expected Experience

Keyboard focus should be visible properly for Custom Domain, Allow protected append writers text box.

### User Impact:
Keyboard users will face difficulty if Keyboard focus is not visible properly for Custom Domain, Allow protected append writers text box.
###Attachment:
![A11y_Azure Tools Storage Explorer_Properties_Blob Container Properties tab_Keyboard](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/73c678c8-274e-423f-be0a-340e44786ae8)

**Comment by MRayermannMSFT** [comment]
An outline will now be visible on the inputs so focus can be visibly tracked:
![7623](https://github.com/microsoft/AzureStorageExplorer/assets/21112697/8c687466-51b5-4895-a11f-e7f3a04b09d5)

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Microsoft Azure Storage Explorer
version: 1.33.0-dev(96)
Issue is fixed. hence closing the bug Please find added attachment for reference.
![7623_Fixed](https://github.com/microsoft/AzureStorageExplorer/assets/98530069/f026b4e7-0893-48bf-9120-eb0353520bfc)
